src/cdk/menu/menu-trigger.spec.ts

```diff
@@ -602,3 +656,20 @@ class StandaloneTriggerWithInlineMenu {
 class TriggerWithData {
   menuData: unknown;
 }
+
+@Component({
+  template: `
+    <button [cdkMenuTriggerFor]="null">First</button>
```

Comment:
I don't think that we need an entirely separate fixture where it is set to null. Instead we should adapt an existing fixture so we can also test that the DOM is updated on dynamic changes. See how it's done for `MatMenu` https://github.com/angular/components/blob/main/src/material/menu/menu.spec.ts#L92.

Reply:
That's good idea, I updated tests.